lib/core-complete.sh

```diff
@@ -2560,6 +2560,9 @@ function ble/complete/source:command/.print-command {
     #   の補完を実行すると遅くなる可能性がある。
     local slow_compgen=
     if [[ ! $COMPV ]]; then
+      shopt -q no_empty_cmd_completion && return 0
+      slow_compgen=1
+    elif [[ :$PATH: == *:/mnt/*:* && ble/base/is-wsl ]]; then
```

Comment:
```suggestion
    elif [[ :$PATH: == *:/mnt/*:* ]] && ble/base/is-wsl; then
```

I'm afraid to say that `[[ ble/base/is-wsl ]]` wouldn't work as you expect. `[[ ... ]]` and `ble/base/is-wsl` are separate commands. They are syntactically on the equivalent level, so one cannot include `ble/base/is-wsl` inside `[[ ... ]]`. `[[ ble/base/is-wsl ]]` is interpreted by `[[ ]]` as `[[ <string> ]]` (which checks whether the `<string>` is empty), so it always evaluates to "true".

Reply:
Thank you for detailed explanation. I fixed it and understand interpretation of `[[ ble/base/is-wsl ]]`.
I need to reread man...